app/namespace.go

```diff
@@ -145,6 +186,33 @@ func (c *NamespaceClient) parseExistingCerts(ctx *cli.Context) (namespace *names
 	return n, existingCerts, nil
 }
 
+func (c *NamespaceClient) toUserNamespacePermissions(userPermissionsInput map[string]string) ([]*auth.UserNamespacePermissions, error) {
+	var res []*auth.UserNamespacePermissions
+	for email, actionGroup := range userPermissionsInput {
+		u, err := c.authClient.GetUser(c.ctx, &authservice.GetUserRequest{
```

Comment:
is this potentially going to cause any rate limiting by the API?

Reply:
it could but what options do we have here? some kind of artificial sleep?

Reply:
agreed offline to move this fetching of users to the server side as part of another pr.